static/js/services/vulnerability-service.js

```diff
@@ -502,14 +542,14 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
       'description': 'This metric measures the impact on confidentiality of a successfully exploited vulnerability. Confidentiality refers to limiting information access and disclosure to only authorized users, as well as preventing access by, or disclosure to, unauthorized ones. Increased confidentiality impact increases the vulnerability score.',
       'values': [
         {
-          'id': 'C',
-          'title': 'Complete',
+          'id': 'H',
+          'title': 'High',
```

Comment:
Some of these descriptions have been modified in the v3 spec i.e. this one:
```
There is total information disclosure, resulting in all system files being revealed. The attacker is able to read all of the system\'s data (memory, files, etc.)
```
to
```
There is a total loss of confidentiality, resulting in all resources within the impacted component...
```
I assume we want to keep these aligned?

Reply:
Sure, I will keep the descriptions aligned.